backend/src/main/java/com/bakdata/conquery/models/query/queryplan/aggregators/specific/EventDateUnionAggregator.java

```diff
@@ -22,7 +22,7 @@
  *
  */
 @RequiredArgsConstructor
-public class EventDateUnionAggregator implements Aggregator<Collection<CDateRange>> {
+public class EventDateUnionAggregator implements Aggregator<CDateSet>{
```

Comment:
Und das hier ist der von den Usern ausgewählte oder?

Reply:
Genau, der kann von den Usern im Frontend gewählt werden aber er wird auch auch automatisch gesetzt in CQConcept, wenn eine DataAggregationMode gesetzt ist oder das Concept nicht von den Dates excluded werden soll